### PR TITLE
Added replicas data to topic overview page

### PR DESCRIPTION
**Pull request by lazzy-panda**
**What changes did you make?** (Give an overview)
Added replicas data to the table on the topic overview page

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

image

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx [hunk 1]
```diff
@@ -100,21 +101,32 @@ const Overview: React.FC<Props> = ({
...
+            {partitions?.map((partition: Partition) => (
+              <tr key={`partition-list-item-key-${partition.partition}`}>
+                <td>{partition.partition}</td>
+                <td>
+                  {partition.replicas?.map((replica: Replica, i: number) => (
+                    <ReplicaCell
+                      leader={replica.leader}
+                      key={`replica-list-item-key-${replica.broker}`}
+                    >
+                      {i + 1 !== partition.replicas?.length
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:121]:
```suggestion
                      {i !== partition.replicas?.length - 1
```

Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:121]:
```suggestion
                      {i !== partition.replicas?.length - 1
```

Comment (workshur) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx:121]: Or you can use pure css solution like 

```
${ReplicaCell} ~ ${ReplicaCell}::before {
  content: ', '; 
}
```

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx [hunk 2]
```diff
@@ -66,6 +66,17 @@ describe('Overview', () => {
...
 
+  it('at least one replica was rendered', () => {
+    setupComponent({
+      ...defaultProps,
+      partitions: mockPartitions,
+      underReplicatedPartitions: 0,
+      inSyncReplicas: 1,
+      replicas: 2,
+    });
+    expect(screen.getAllByLabelText('replica-info')).toHaveLength(2);
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx:77]: > at least one replica was rendered

But you expect 2

Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx:77]: > at least one replica was rendered

But you expect 2

Comment (workshur) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx:77]: Please check that you have correct values of Replica-info

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts [hunk 3]
```diff
@@ -111,6 +111,12 @@ export const Preview = styled.span`
   cursor: pointer;
 `;
 
+export const ReplicaCell = styled.span.attrs({ 'aria-label': 'replica-info' })<{
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts:114]: Why are we adding ReplicaCell to common component? It must leave in topics section

Comment (workshur) [inline kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts:114]: Why we adding ReplicaCell to common component?

File: kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx [hunk 4]
```diff
@@ -35,6 +36,15 @@ describe('TableHeaderCell', () => {
...
   });
 
+  it('renders replica cell with props', () => {
+    render(<ReplicaCell leader />);
+    expect(screen.getByLabelText('replica-info')).toBeInTheDocument();
+    expect(screen.getByLabelText('replica-info')).toHaveStyleRule(
+      'color',
+      'orange'
+    );
+  });
```
Comment (workshur) [inline kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx:46]: How does this test relate to tables?

Comment (workshur) [inline kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx:46]: How does this test relate to tables?

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx (hunk 1 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx]: Implemented

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx (hunk 2 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx]: Fixed

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts (hunk 3 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts]: Moved to Topics

File: kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx (hunk 4 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/common/table/__tests__/TableHeaderCell.spec.tsx]: Fixed

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx (hunk 2 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx]: fixed

**Review by workshur — APPROVED** [review]

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx [hunk 5]
```diff
@@ -66,6 +67,25 @@ describe('Overview', () => {
...
+      underReplicatedPartitions: 0,
+      inSyncReplicas: 1,
+      replicas: 1,
+    });
+    expect(screen.getByLabelText('replica-info')).toBeInTheDocument();
+  });
+
+  it('renders replica cell with props', () => {
+    render(<ReplicaCell leader />);
+    expect(screen.getByLabelText('replica-info')).toBeInTheDocument();
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx:82]: i think element repeats twice we can put it in a variable

**Review by Mgrdich — APPROVED** [review]

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx (hunk 5 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx]: fixed

**Pull request merged**